components/testimonials/index.js

```diff
@@ -0,0 +1,121 @@
+import Glide from '@glidejs/glide'
+import { rem } from 'polished'
+import { useEffect } from 'react'
+import styled from 'styled-components'
+import Slide from './slide'
+import '@glidejs/glide/dist/css/glide.core.min.css'
+import '@glidejs/glide/dist/css/glide.theme.min.css'
+
+const DOT_SIZE = rem('18px')
+const DOT_SIZE_MOBILE = rem('24px')
+
+const Root = styled.div`
```

Comment:
Maybe we want to move styles to `components/testimonials/styles.js` 

What you guys think? @superkhau @timothy-wan

Reply:
Well current rest of the app and styled components are laid out like this anyway. I'm good leaving like this